client/app/components/ParameterMappingInput.jsx

```diff
@@ -336,15 +339,15 @@ class MappingEditor extends React.Component {
 
   renderContent() {
     const { mapping, inputError } = this.state;
-    const helpUrl = replace(HELP_URL, '{0}', 'edit_mapping');
+    const [helpUrl, tooltip] = HELP_URL;
 
     return (
       <div className="parameter-mapping-editor">
         <header>
           Edit Source and Value
           {/* eslint-disable-next-line react/jsx-no-target-blank */}
           <a href={helpUrl} target="_blank" rel="noopener">
-            <Tooltip title="Learn more about editing query paramaters (opens in a new window)">
+            <Tooltip title={tooltip}>
```

Comment:
Just a thought: maybe we should have a `<HelpTrigger>` component, that currently will: render circle icon with a tooltip. We will use it in all the places we add help content to. But later on allow us in one single swoop change the implementation to open a [drawer](https://ant.design/components/drawer/) with the help content inside the app?

(not a blocker of course, I'm waiting for the relevant docs PR to merge)

Reply:
I like the drawer idea. Will the content be retrieved with github api by help url?

Reply:
No, I thought we will update the Gatsby configuration to create a "drawer friendly" view of the help content alongside the regular one. The drawer pages will have the same URL structure, so we can deduce the new URL based on the regular one (so Cmd/Shift+Click will still work).